csrankings-p.csv

```diff
@@ -23,7 +23,8 @@ P. Vassalos,AUEB,http://pubs.dbs.uni-leipzig.de/aa/aff/Greece/Athens/Athens%20Un
 P. Vijay Kumar,IISc Bangalore,http://ece.iisc.ernet.in/~vijay,Wx5neQQAAAAJ
 Pabitra Mitra,IIT Kharagpur,http://cse.iitkgp.ac.in/~pabitra,5bXSZPYAAAAJ
 Pablo A. Parrilo,Massachusetts Institute of Technology,http://www.mit.edu/~parrilo,s_IPfEYAAAAJ
-Pablo Alejandro Figueroa,Universidad de los Andes,https://profesores.virtual.uniandes.edu.co/pfiguero/es/inicio,1pVmx_4AAAAJ
+Pablo Alejandro Figueroa,Universidad de los Andes,https://profesores.virtual.uniandes.edu.co/pfiguero/es/inicio,1pVmx_4AAAAJ
+Pablo Arbeláez,Universidad de los Andes,https://biomedicalcomputervision.uniandes.edu.co,k0nZO90AAAAJ

```

Comment:
Please clarify relationship with el Departamento de Ingeniería de Sistemas y Computación in light of the inclusion criteria.

Reply:
Pablo Arbeláez have advised 2 PhD graduates in CS and AI, and is currently an advisor to 4 PhD students in CS and AI. Additionally, he is the director of the Center for Research and Formation in Artificial Intelligence ([CINFONIA](https://cinfonia.uniandes.edu.co)) at Universidad de los Andes.

Reply:
So just to be clear: these PhD students are going to graduate with a degree granted by DISC?

Reply:
In our University the PhD title is granted by the School of Engineering. These students will graduate with a PhD in Engineering, the same title as DISC students.